### ISSUE TITLE
receiving ascii codec when attempting to fetch entries

### ISSUE THREAD

**Issue by jarkyll** [contentful/contentful.py#4]
After successfully connecting to the client, there is an issue running client.entries() where I get an error. 
"UnicodeEncodeError: 'ascii' codec can't encode character u'\u2019' in position 189: ordinal not in range(128)"

**Comment by dlitvakb** [comment]
Hey @jarkyll,

Could you provide me your Python version, and the content that may be producing this?

Cheers

**Comment by dlitvakb** [comment]
Hey @jarkyll,

Could you provide me your Python version, and the content that may be producing this?

Cheers

**Comment by nickdandakis** [comment]
@dlitvakb, I'm getting a similar error.
`UnicodeEncodeError: 'ascii' codec can't encode character u'\u2014' in position 98: ordinal not in range(128)`

Python 2.7.13. Specifically trying to get some entries from within a very basic Flask application.

Full stack
```
Traceback (most recent call last):
  File "/Users/ndandakis/Workspace/digest/venv/lib/python2.7/site-packages/flask/app.py", line 1994, in __call__
    return self.wsgi_app(environ, start_response)
  File "/Users/ndandakis/Workspace/digest/venv/lib/python2.7/site-packages/flask/app.py", line 1985, in wsgi_app
    response = self.handle_exception(e)
  File "/Users/ndandakis/Workspace/digest/venv/lib/python2.7/site-packages/flask/app.py", line 1540, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/ndandakis/Workspace/digest/venv/lib/python2.7/site-packages/flask/app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/ndandakis/Workspace/digest/venv/lib/python2.7/site-packages/flask/app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/ndandakis/Workspace/digest/venv/lib/python2.7/site-packages/flask/app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/ndandakis/Workspace/digest/venv/lib/python2.7/site-packages/flask/app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/ndandakis/Workspace/digest/venv/lib/python2.7/site-packages/flask/app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/ndandakis/Workspace/digest/server.py", line 10, in index
    posts = contentful_client.entries({'content_type': 'post'});
  File "/Users/ndandakis/Workspace/digest/venv/lib/python2.7/site-packages/contentful/client.py", line 223, in entries
    query
  File "/Users/ndandakis/Workspace/digest/venv/lib/python2.7/site-packages/contentful/client.py", line 462, in _get
    response.json()
  File "/Users/ndandakis/Workspace/digest/venv/lib/python2.7/site-packages/contentful/resource_builder.py", line 48, in build
    return self._build_array()
  File "/Users/ndandakis/Workspace/digest/venv/lib/python2.7/site-packages/contentful/resource_builder.py", line 62, in _build_array
    ) for item in self.json['items']]
  File "/Users/ndandakis/Workspace/digest/venv/lib/python2.7/site-packages/contentful/resource_builder.py", line 83, in _build_item
    depth=self.depth
  File "/Users/ndandakis/Workspace/digest/venv/lib/python2.7/site-packages/contentful/resource.py", line 84, in __init__
    self._fields = self._hydrate_fields(item, localized, includes)
  File "/Users/ndandakis/Workspace/digest/venv/lib/python2.7/site-packages/contentful/resource.py", line 112, in _hydrate_fields
    includes
  File "/Users/ndandakis/Workspace/digest/venv/lib/python2.7/site-packages/contentful/entry.py", line 50, in _coerce
    return content_type_field.coerce(value)
  File "/Users/ndandakis/Workspace/digest/venv/lib/python2.7/site-packages/contentful/content_type_field.py", line 37, in coerce
    return self._coercion.coerce(value)
  File "/Users/ndandakis/Workspace/digest/venv/lib/python2.7/site-packages/contentful/content_type_field_types.py", line 47, in coerce
    return str(value)
```

I'm pretty sure this is happening because of the lack of `utf-8` encoding when coercing values to strings in `content_type_field_types.py`

**Comment by dlitvakb** [comment]
Hey @nickdandakis,

Yup, it looks like that, could you send me a little bit of the content of an entry that is triggering this error so that I can add a test around it?

Cheers

**Comment by nickdandakis** [comment]
Unfortunately I can't for legal reasons. I don't see any content that blatantly needs `utf-8`.

**Comment by dlitvakb** [comment]
Ok, then I'll just try a random test with a single emoji and see if that breaks it.

Cheers

**Comment by nickdandakis** [comment]
@dlitvakb, did some more digging. The following unicode characters are making line 47 of `content_type_field_types.py` fail.

```
\u2014
\u2019
\u201c
\u201d
```

```
EM DASH —
RIGHT SINGLE QUOTATION MARK ’
LEFT DOUBLE QUOTATION MARK “
RIGHT DOUBLE QUOTATION MARK ”
```

I'm sure it'll fail for all unicode characters, but I hope this is helpful!

**Comment by dlitvakb** [comment]
@nickdandakis very helpful, thanks 👍

**Comment by dlitvakb** [comment]
@nickdandakis @jarkyll Fix included in #5.

Will get it merged and released as soon as possible.

Cheers

**Comment by dlitvakb** [comment]
Release 1.0.3 just pushed to PyPI.

Cheers
